generated/nidigital/nidigital_service.cpp

```diff
@@ -205,6 +224,8 @@ namespace nidigital_grpc {
       auto vi_grpc_session = request->vi();
       ViSession vi = session_repository_->access_session(vi_grpc_session.id(), vi_grpc_session.name());
       session_repository_->remove_session(vi);
+      auto status = library_->Close(vi);
```

Comment:
@ShantanuShinde this change is not relevant to this PR and can cause confusion for review. So consider undoing this change here. @ni-pkhare  will be sending out a separate PR for this change.

Reply:
removed them